install.sh

```diff
@@ -106,18 +102,24 @@ verify_configuration_file() {
     log "${INVALID_CONTENT}"
     HAS_VALIDATION_ERR=1
   fi
-
-  if [ -z "${POPULATED_LICENSE}" ];  then
-    if [ -z "${TF_VAR_bamboo_license}" ]; then
-      log "License is missing. Please provide license in config file, or export it to the environment variable 'TF_VAR_bamboo_license'." "ERROR"
-      HAS_VALIDATION_ERR=1
+  INSTALLED_BAMBOO=$(get_product "bamboo" "${CONFIG_ABS_PATH}")
+  if [ -n "${INSTALLED_BAMBOO}" ]; then
+    # check license and admin password
+    export POPULATED_LICENSE=$(grep -o '^[^#]*' "${CONFIG_ABS_PATH}" | grep 'bamboo_license')
+    export POPULATED_ADMIN_PWD=$(grep -o '^[^#]*' "${CONFIG_ABS_PATH}" | grep 'bamboo_admin_password')
+
+    if [ -z "${POPULATED_LICENSE}" ];  then
+      if [ -z "${TF_VAR_bamboo_license}" ]; then
+        log "License is missing. Please provide Bamboo license in config file, or export it to the environment variable 'TF_VAR_bamboo_license'." "ERROR"
+        HAS_VALIDATION_ERR=1
+      fi
     fi
-  fi
 
-  if [ -z "${POPULATED_ADMIN_PWD}" ];  then
-    if [ -z "${TF_VAR_bamboo_admin_password}" ]; then
-      log "Admin password is missing. Please provide admin password in config file, or export it to the environment variable 'TF_VAR_bamboo_admin_password'." "ERROR"
-      HAS_VALIDATION_ERR=1
+    if [ -z "${POPULATED_ADMIN_PWD}" ];  then
```

Comment:
Can these nested `if` blocks be simplified like so

```shell
if [ -z "${POPULATED_ADMIN_PWD}" ]  && [ -z "${TF_VAR_bamboo_admin_password}" ]
then
   log "Admin password is missing. Please provide Bamboo admin password in config file, or export it to the environment variable 'TF_VAR_bamboo_admin_password'." "ERROR"
        HAS_VALIDATION_ERR=1
fi
```

Reply:
Thanks for pointing this. Yes, we definitely can simplify this.

Done